fixmyapp/serializers.py

```diff
@@ -63,7 +63,8 @@ class Meta:
 
 class ProjectSerializer(serializers.HyperlinkedModelSerializer):
     faq = QuestionSerializer(many=True)
-    photos = PhotoSerializer(many=True, default=[Photo(**PLACEHOLDER_PHOTO)])
+    # photos = PhotoSerializer(many=True, default=[Photo(**PLACEHOLDER_PHOTO)])
+    photos = PhotoSerializer(many=True)
```

Comment:
removing the `default` argument outputed a correct `YAML` file for the schema

Reply:
Do you know if this has any sideeffects?